roles/6-generic-apps/tasks/main.yml

```diff
@@ -69,6 +69,12 @@
   when: wordpress_install
   tags: wordpress
 
+- name: GITEA
```

Comment:
Let's make `roles/6-generic-apps/tasks/main.yml` alphabetical if possible, please pushing Gitea up closer to the top as it begins with the letter "g"

Reply:
Should I move MediaWiki as well?

Reply:
Nah, leave MediaWiki next to DokuWiki, as they're very closely related, so it's ok to break the alphabetical order ocassionally.